src/internal/mod.rs

```diff
@@ -82,3 +83,10 @@ pub fn recv_from(socket: CSocket,
         Ok(len as usize)
     }
 }
+
+pub fn duration_to_timeval(dur: Duration) -> libc::timeval {
```

Comment:
This function doesn't build on Windows due to type mismatches - check the build output for AppVeyor to see what's going on.